controller/controller.php

```diff
@@ -44,6 +44,7 @@
         include '../View/includes/head.php';
         include '../View/includes/navbar.php';
         include '../view/home/home.php';
+        getAllColumbariumRelatedDataWithFilterTest();
```

Comment:
Always remove test calls from controller when they are done. Calling test methods from the controller is not the best way to do testing, but for simplicity we will keep calling test methods from the controller and then removing the calls.